pkg/proc/arm64_arch.go

```diff
@@ -13,16 +13,23 @@ import (
 )
 
 var arm64BreakInstruction = []byte{0x0, 0x0, 0x20, 0xd4}
+var arm64WindowsBreakInstruction = []byte{0x0, 0x0, 0x3e, 0xd4}
```

Comment:
Can you add some documentation context for why Windows uses a different breakpoint instruction, please.

Reply:
Done!